google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -391,6 +396,12 @@ private Object writeReplace() {
           case JSON:
             builder.set(fieldName).to(Value.json((String) value));
             break;
+          case PROTO:
+            builder
+                .set(fieldName)
+                .to(Value.protoMessage((ByteArray) value, fieldType.getProtoTypeFqn()));
```

Comment:
Add `break` statement
```suggestion
                .to(Value.protoMessage((ByteArray) value, fieldType.getProtoTypeFqn()));
                break;
```